coderadar-core/src/main/java/io/reflectoring/coderadar/projectadministration/domain/Project.java

```diff
@@ -1,9 +1,11 @@
 package io.reflectoring.coderadar.projectadministration.domain;
 
+import io.reflectoring.coderadar.useradministration.domain.Team;
 import java.util.Date;
+import java.util.List;
 import lombok.Data;
 
-/** A coderadar project that defines the source of files that are to be analyzed. */
+/** A Coderadar project that defines the source of files that are to be analyzed. */
```

Comment:
shouldn't it be coderadar with a lower case c?

Reply:
Why do you think so?